jruby/src/main/java/org/kohsuke/stapler/jelly/jruby/RackDispatcher.java

```diff
@@ -32,24 +32,29 @@ public boolean dispatch(final RequestImpl req, ResponseImpl rsp, Object node) th
             return false;
 
         // TODO: does the context need to live longer?
-        ServletRackContext rackContext = new DefaultServletRackContext(new ServletRackConfig(req.getServletContext()));
+        // TODO needs triage
+        //ServletRackContext rackContext = new DefaultServletRackContext(new ServletRackConfig(req.getServletContext()));
+        ServletRackContext rackContext = null;
 
         // we don't want the Rack app to consider the portion of the URL that was already consumed
         // to reach to the Rack app, so for PATH_INFO we use getRestOfPath(), not getPathInfo()
-        ServletRackEnvironment env = new ServletRackEnvironment(req, rsp, rackContext) {
-            @Override
-            public String getPathInfo() {
-                return req.getRestOfPath();
-            }
-        };
+        // TODO needs triage
+        ServletRackEnvironment env = null;
+        //ServletRackEnvironment env = new ServletRackEnvironment(req, rsp, rackContext) {
+        //    @Override
+        //    public String getPathInfo() {
+        //        return req.getRestOfPath();
+        //    }
+        //};
         // servletHandler = Rack::Handler::Servlet.new(node)
         runtime.getLoadService().require("rack/handler/servlet");
         IRubyObject servletHandler = ((RubyModule)runtime.getModule("Rack").getConstantAt("Handler")).getClass("Servlet").callMethod("new", x);
 
         DefaultRackApplication dra = new DefaultRackApplication();
         dra.setApplication(servletHandler);
-        dra.call(env)
-                .respond(new ServletRackResponseEnvironment(Stapler.getCurrentResponse()));
+        // TODO needs triage
+        //dra.call(env)
+        //        .respond(new ServletRackResponseEnvironment(Stapler.getCurrentResponse()));
```

Comment:
Hopefully we delete JRuby support before we ever have to deal with these TODOs.

Reply:
bin it 🎉